kms/pom.xml

```diff
@@ -64,6 +63,16 @@
       <version>1.1.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>com.nimbusds</groupId>
+        <artifactId>nimbus-jose-jwt</artifactId>
+        <version>9.31</version>
+    </dependency>
+    <dependency>
```

Comment:
It doesn't look like this dependency is being used. Please remove.

Reply:
Hi, thanks for looking at this.
Actually,  "nimbus-jose-jwt" is a dependency being used in this update.
It is used in java-docs-samples/kms/src/main/java/kms/ConvertPublicKeyToJwk.java

Reply:
Right, I'm referring to the `org.bouncycastle:bcpkix-jdk15on` dependency L71-75

Reply:
That's also needed by the nimbus-jose-jwt.
When I remove that dependency, I see this error:
![Screenshot 2023-04-20 at 9 42 29 AM](https://user-images.githubusercontent.com/32246504/233385598-395f2273-ca31-4b10-ba4c-65eb62ab0178.png)

Reply:
Sorry about that. Thanks!